Makefile

```diff
@@ -33,14 +33,58 @@ help:
 
 .PHONY: help
 
+# these are various tools required by make commands below
+GH := $(shell command -v gh 2> /dev/null)
+PYTHON := $(shell command -v python 2> /dev/null)
+JQ := $(shell command -v jq 2> /dev/null)
```

Comment:
These are tools required to be present in development environments for make commands to work. This change checks for said tools in advance to fail early and prompt devs to install what's needed. 

GitHub action environments come with those tools pre-installed.